### PR TITLE
Specter 2: Include frame ID in all logs for use in analytics charts.

### PR DESCRIPTION
**Pull request by lynnerang**
Will be helpful to be able to break stats down by frame, e.g. how many of each type of annotations tend to be added to a frame.

### CODE REVIEW

File: specter2/src/code.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
   ANNOTATION_LISTS,
   PLUGIN_INFO,
 } from './constants';
+import { findTopFrame } from '../node_modules/linkedin-design-tools-figma/dist/index';
```
Comment (gblakeman) [inline specter2/src/code.ts:13]: You should be able to pull shared tools stuff in via `require` without hard-coding a path:

```ts
const { findTopFrame } = require('linkedin-design-tools-figma');
```

**Review by gblakeman — APPROVED** [review]
🌈🦄

File: specter2/src/code.ts (hunk 1 above)
Comment (lynnerang) [reply specter2/src/code.ts:13]: ah thanks for catching, it does this when I use it before writing the import and VS code auto-fixes

Comment (gblakeman) [reply specter2/src/code.ts]: Ha, that’s kinda cool. I don’t think I have that VS Code extension.

**Pull request merged**